proto/api/v1/deployment_service.proto

```diff
@@ -74,4 +83,8 @@ service DeploymentService {
   rpc GetLabels(Empty) returns (DeploymentLabelsResponse) {
     option (google.api.http) = {get: "/v1/deployments/metadata/labels"};
   }
+
+  rpc Export(ExportDeploymentRequest) returns (stream ExportDeploymentResponse) {
+    option (google.api.http) = {get: "/v1/export/deployments"};
```

Comment:
Should this be `/v1/deployments/export`?  I never remember which order these should go in, but down below, there is `/v1/images/cache/invalidate`.  If that is the correct way then export should be on the end.

Reply:
So...this is not possible because of the /v1/deployments/{id} where export would be picked up by the id

Reply:
Which I should know because just last week I used the same argument on a compliance API.